app/policies/kondo_policy.rb

```diff
@@ -3,5 +3,25 @@ class Scope < Scope
     def resolve
       scope.all
     end
+
+    def new?
+      current_user.provider?
+    end
+
+    def create?
+      current_user.provider?
+    end
+
+    def edit?
+      current_user.id == kondo.user_id
+    end
```

Comment:
This code is not needed, reason being is that inside the application_policy.rb the `edit?` method calls the `update?` method from this file. Would be happy to explain more if its not clear :)